endpoints-control/src/test/java/com/google/api/control/ControlFilterTest.java

```diff
@@ -26,6 +26,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+import autovalue.shaded.com.google.common.common.collect.Sets;
```

Comment:
yeah I noticed it. I just I'd prefer cleaning up those shallowed packages in a separate PR. This PR only reorders import.